tests/robot-tests/tests/admin_and_public/bau/data_catalogue.robot

```diff
@@ -135,3 +143,8 @@ Check page displays 'This is not the latest data' tag
 Select new subject and download new subject file
     user clicks checkbox    ${SUBJECT_NAME_1} (csv, 456 Kb)
     user clicks button    Download selected files
+
+Validate new zip contains correct files
+    sleep    5    # wait for file to download
```

Comment:
we potentially may want to pass a timeout to the `zip should contain directories and files` instead of setting a hard-coded sleep in order to avoid transient errors when this is running in a CI environment. We would then raise an assertion error inside `zip should contain directories and files` if the file isn't downloaded within the expected time

Reply:
On second thoughts, we probably want to keep the sleep here but 15 seconds is a more lenient time, especially when testing against slower environments.